net-p2p/go-ipfs-bin/files/ipfs.init

```diff
@@ -0,0 +1,27 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+IPFS_PATH=${IPFS_PATH:-/var/lib/ipfs/.ipfs}
+
+depend() {
+	need net
+}
+
+start() {
```

Comment:
Please consider using the various openrc variables instead of caling start-stop-daemon directly (possibly leaving out start() and stop(). A nice example: https://github.com/gentoo/gentoo/blob/master/dev-db/redis/files/redis.initd-5